### ISSUE TITLE
Addon jar contains duplicate class files

### ISSUE THREAD

**Issue by johndevs** [jouni/Animator#9]
The addon jar contains an extra package "build.classes.org.vaadin.jouni.animator" with duplicates of all the class files.

Using animator jar version 1.7.1

**Comment by thjaeckle** [comment]
Hi.
This could be related to what I am experiencing. Tomcat tries to parse the .jar file and prints out following WARNINGS:
(Using version 1.6.6)

```
WARNING: Unable to load class [org.vaadin.jouni.animator.AnimatorApplication$7] to check against the @HandlesTypes annotation of one or more ServletContentInitializers.
java.lang.ClassNotFoundException: org.vaadin.jouni.animator.AnimatorApplication$7
        at org.apache.catalina.loader.WebappClassLoader.loadClass(WebappClassLoader.java:1688)
        at org.apache.catalina.loader.WebappClassLoader.loadClass(WebappClassLoader.java:1533)
        at org.apache.catalina.startup.ContextConfig.checkHandlesTypes(ContextConfig.java:1988)
        at org.apache.catalina.startup.ContextConfig.processAnnotationsStream(ContextConfig.java:1951)
        at org.apache.catalina.startup.ContextConfig.processAnnotationsJar(ContextConfig.java:1840)
        at org.apache.catalina.startup.ContextConfig.processAnnotationsUrl(ContextConfig.java:1808)
        at org.apache.catalina.startup.ContextConfig.processAnnotations(ContextConfig.java:1794)
        at org.apache.catalina.startup.ContextConfig.webConfig(ContextConfig.java:1214)
        at org.apache.catalina.startup.ContextConfig.configureStart(ContextConfig.java:828)
        at org.apache.catalina.startup.ContextConfig.lifecycleEvent(ContextConfig.java:302)
        at org.apache.catalina.util.LifecycleSupport.fireLifecycleEvent(LifecycleSupport.java:119)
        at org.apache.catalina.util.LifecycleBase.fireLifecycleEvent(LifecycleBase.java:90)
        at org.apache.catalina.core.StandardContext.startInternal(StandardContext.java:5148)
        at org.apache.catalina.util.LifecycleBase.start(LifecycleBase.java:150)
        at org.apache.catalina.core.ContainerBase.addChildInternal(ContainerBase.java:866)
        at org.apache.catalina.core.ContainerBase.addChild(ContainerBase.java:842)
        at org.apache.catalina.core.StandardHost.addChild(StandardHost.java:615)
        at org.apache.catalina.startup.HostConfig.deployDirectory(HostConfig.java:1095)
        at org.apache.catalina.startup.HostConfig$DeployDirectory.run(HostConfig.java:1617)
        at java.util.concurrent.Executors$RunnableAdapter.call(Executors.java:441)
        at java.util.concurrent.FutureTask$Sync.innerRun(FutureTask.java:303)
        at java.util.concurrent.FutureTask.run(FutureTask.java:138)
        at java.util.concurrent.ThreadPoolExecutor$Worker.runTask(ThreadPoolExecutor.java:886)
        at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:908)
        at java.lang.Thread.run(Thread.java:662)
```

In one package the AnimatorApplication is present, but in the other package it isn't.
The Application should not be part of the .jar file anyway, or should it?

Regards
TJ

**Comment by jouni** [comment]
No, the demo application shouldn't be a part of the JAR. I've just packaged it hastily (I don't have a proper build script, just using Eclipse export) and forgot some additional folders/sources there.

**Comment by jouni** [comment]
Hopefully fixed in 1.7.4 (coming shortly to Vaadin Directory).

1.7.4 built from 6e62bca34b0d80df70920db62f259840d149ceda
